docbook/reference/en/en-US/modules/RESTEasy_Spring_Integration.xml

```diff
@@ -507,29 +741,28 @@ import jakarta.ws.rs.core.Application;
 @Component
 @ApplicationPath("/sample-app/")
 public class JaxrsApplication extends Application {
-}]]></programlisting>
+}]]>
+            </programlisting>
         </para>
         <para>
-            Finally, to register &REST-API; resources and providers, just define them as Spring beans, and they will be
+            To register &REST-API; resources and providers, define them as Spring beans.  They will be
             automatically registered. Notice that &REST-API; resources can be singleton or request scoped, while &REST-API;
             providers must be singletons.
         </para>
-        <para>To see complete examples, please check the <ulink
+        <para>To see an example, please check the <ulink
                 url="https://github.com/resteasy/resteasy-spring-boot/tree/master/sample-app">sample-app
-        </ulink> in the project codebase.
+        </ulink>.
         </para>
     </section>
+
     <section id="SpringUpgradeInWildfly">
         <title>Upgrading in WildFly</title>
         <para>
-            <emphasis role="bold">Note.</emphasis>
-            As noted in<xref linkend="upgrading-wildfly"/>, the RESTEasy distribution comes with a zip file
-            called <code><![CDATA[resteasy-jboss-modules-<version>.zip]]></code>,
-            which can be unzipped into the modules/system/layers/base/ directory of WildFly to upgrade to a new version
-            of RESTEasy.
-            Because of the way resteasy-spring is used in WildFly, after unzipping the zip file, it is also necessary to
-            remove the
-            old resteasy-spring jar from
+            As noted in <xref linkend="upgrading-wildfly"/>, the RESTEasy distribution comes with a zip file
+            called <code><![CDATA[resteasy-jboss-modules-<version>.zip]]></code>.
```

Comment:
I don't think this is true anymore since we use Galleon feature packs.

Reply:
maybe these blog posts can be used for reference:

- [Using The RESTEasy Galleon Feature Pack In WildFly](https://resteasy.dev/2021/11/22/feature-pack/)
- [Deploying `resteasy-spring` based Project Into WildFly Full Distribution](https://resteasy.dev/2021/10/13/resteasy-spring-wildfly-and-more/)
- [Deploy RESTEasy-Spring project into WildFly Servlet-Only Container](https://resteasy.dev/2020/04/10/spring/)